boostlook.css

```diff
@@ -1097,3 +1186,226 @@ body > .chapter,
     top: 0.2rem;
   }
 }
+
+/* ---------------- Styling from boostbookV2.css ----------------*/
+
+body[bgcolor="white"]:has(.boostlook):not(:has(.doc)) {
+  background-color: transparent;
+}
+
+.boostlook:not(:has(.doc)) {
+  border-radius: 0;
+}
+/* general elements */
+.boostlook:not(:has(.doc)) h1,
+.boostlook:not(:has(.doc)) h2,
+.boostlook:not(:has(.doc)) h3,
+.boostlook:not(:has(.doc)) h4,
+.boostlook:not(:has(.doc)) h5,
+.boostlook:not(:has(.doc)) h6 {
+  margin: 1em 0em 0.5em;
+}
+
+.boostlook:not(:has(.doc)) h5 {
+  font-style: italic;
+  font-weight: normal;
+  font-size: 110%;
+}
+
+.boostlook:not(:has(.doc)) h1 a,
+.boostlook:not(:has(.doc)) h2 a,
+.boostlook:not(:has(.doc)) h3 a,
+.boostlook:not(:has(.doc)) h4 a {
+  color: var(--bl-header-color)
+}
+
+.boostlook:not(:has(.doc)) p {
+  padding: 1em 0 !important;
+}
+
+.boostlook:not(:has(.doc)) > hr {
+  display: none;
+}
+
+.boostlook:not(:has(.doc)) ol {
+  padding-left: 1em;
+}
+
+.boostlook:not(:has(.doc)) ol li {
+  list-style-type: decimal;
+}
+
+.boostlook:not(:has(.doc)) pre {
+  background-color: var(--bl-pre-background);
+  border: 1px solid var(--bl-code-border-color);
+  border-radius: 8px;
+  padding: 1.2em .8em
+}
```

Comment:
Let's add a margin declaration to address horizontal spacing for pre tags

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/a6fea309-6b93-4d42-9d22-dcb28f431cb2">

Reply:
this should be fixed in the latest commit